dist/umd/image-resize.min.js

```diff
@@ -1,851 +1 @@
removeEventListener("keyup",o.checkImage),o.quill.root.removeEventListener("input",o.checkImage),o.setUserSelect(""))},this.repositionElements=function(){if(o.overlay&&o.img){var e=o.quill.root.parentNode,t=o.img.getBoundingClientRect(),n=e.getBoundingClientRect();Object.assign(o.overlay.style,{left:t.left-n.left-1+e.scrollLeft+"px",top:t.top-n.top+e.scrollTop+"px",width:t.width+"px",height:t.height+"px"})}},this.hide=function(){o.hideOverlay(),o.removeModules(),o.img=void 0},this.setUserSelect=function(e){["userSelect","mozUserSelect","webkitUserSelect","msUserSelect"].forEach(function(t){o.quill.root.style[t]=e,document.documentElement.style[t]=e})},this.checkImage=function(e){window.Quill&&o.img&&(46!=e.keyCode&&8!=e.keyCode||window.Quill.find(o.img).deleteAt(0),o.hide())},this.quill=t,this.options=l(l({},n),s.a),document.execCommand("enableObjectResizing",!1,"false"),this.quill.root.addEventListener("click",this.handleClick,!1),this.quill.root.addEventListener("mscontrolselect",this.handleClick,!1),this.quill.root.addEventListener("scroll",this.handleScroll,!1),this.quill.on("text-change",this.onUpdate),this.quill.root.parentNode.style.position=this.quill.root.parentNode.style.position||"relative",this.moduleClasses=this.options.modules,this.modules=[]};if(t.default=y,window.Quill){var g=["alt","height","width","style"],m=window.Quill.import("formats/image"),b=function(e){function t(){return r(this,t),n(this,(t.__proto__||Object.getPrototypeOf(t)).apply(this,arguments))}return i(t,e),d(t,[{key:"format",value:function(e,o){g.indexOf(e)>-1?o?this.domNode.setAttribute(e,o):this.domNode.removeAttribute(e):p(t.prototype.__proto__||Object.getPrototypeOf(t.prototype),"format",this).call(this,e,o)}}],[{key:"formats",value:function(e){return g.reduce(function(t,o){return e.hasAttribute(o)&&(t[o]=e.getAttribute(o)),t},{})}}]),t}(m);window.Quill.register(b,!0),"function"!=typeof Object.assign&&(Object.assign=function(e){if(null==e)throw new TypeError("Cannot convert undefined or null to object");e=Object(e);for(var t=1;t<arguments.length;t++){var o=arguments[t];if(null!=o)for(var n in o)Object.prototype.hasOwnProperty.call(o,n)&&(e[n]=o[n])}return e}),window.Quill.register("modules/imageResize",y)}},function(e,t){e.exports='<svg viewbox="0 0 18 18">\n  <line class="ql-stroke" x1="15" x2="3" y1="9" y2="9"></line>\n  <line class="ql-stroke" x1="14" x2="4" y1="14" y2="14"></line>\n  <line class="ql-stroke" x1="12" x2="6" y1="4" y2="4"></line>\n</svg>'},function(e,t){e.exports='<svg viewbox="0 0 18 18">\n  <line class="ql-stroke" x1="3" x2="15" y1="9" y2="9"></line>\n  <line class="ql-stroke" x1="3" x2="13" y1="14" y2="14"></line>\n  <line class="ql-stroke" x1="3" x2="9" y1="4" y2="4"></line>\n</svg>'},function(e,t){e.exports='<svg viewbox="0 0 18 18">\n  <line class="ql-stroke" x1="15" x2="3" y1="9" y2="9"></line>\n  <line class="ql-stroke" x1="15" x2="5" y1="14" y2="14"></line>\n  <line class="ql-stroke" x1="15" x2="9" y1="4" y2="4"></line>\n</svg>'}])});
```

Comment:
Looks like this wasn't minified previously (despite the name). Probably ok to do so now?